src/passthrough/inode_store.rs

```diff
@@ -45,16 +52,17 @@ impl InodeStore {
     }
 
     pub fn get_by_ids(&self, ids: &InodeAltKey) -> Option<&Arc<InodeData>> {
-        // safe to unwrap, inode must be in data map if found by ids, otherwise unwrap on
-        // corruption.
-        self.inode_by_ids(ids).map(|inode| self.get(inode).unwrap())
+        match self.inode_by_ids(ids) {
```

Comment:
Can this be simplified as
```
let inode = self.inode_by_ids(ids)?;
self.get(inode)
```

Reply:
using `map`?
```
self.inode_by_ids(ids).map(|id| self.get(id))
```

Reply:
self.inode_by_ids(ids).map(|id| self.get(id)) don't work, self.get(id) return a another Option, not a value